test/DynamoCoreWpfTests/ViewExtensions/PythonMigrationViewExtensionTests.cs

```diff
@@ -228,6 +226,15 @@ public static void RaiseLoadedEvent(FrameworkElement element)
 
             eventMethod.Invoke(element, new object[] { args });
         }
+
+        private void SetupDebugMode()
+        {
+            string configPath = Path.Combine(GetTestDirectory(ExecutingDirectory), "DynamoCoreWpfTests", "python3DebugMode.config");
+            Type dbgModesType = typeof(DebugModes);
+
+            // Load the enabled/disabled status from the test config file.
+            dbgModesType.GetMethod("LoadDebugModesStatusFromConfig", BindingFlags.Static | BindingFlags.NonPublic).Invoke(null, new object[] { configPath });
```

Comment:
Can we make this method internal instead of using reflection?

Reply:
I think so, I would also like to make this function available in `DynamoTestUIBase` so that we do not need to implement it everywhere.. Let me make more changes while waiting for self serve

Reply:
Well, seems it is a one line thing so no need to do that. Changes up soon